src/reporters/errors/decorators/plain-text.js

```diff
@@ -7,7 +11,11 @@ export default {
 
     'code': str => str,
 
-    'code step-source': str => str,
+    'code step-source': str => indentString(str, ' ', CODE_ALIGN_SPACES),
+
+    'code-line': str => `${str}\n`,
```

Comment:
Can we use `<span>` elements for lines with attributes `code-line` and `last-code-line`?
ping @kirovboris